code/game/objects/items/weapons/shields.dm

```diff
@@ -41,13 +41,13 @@
 	. = ..()
 	switch(get_block_chance(user))
 		if(0 to 30)
-			to_chat(user, "So heavy... You feel doubtful in your ability to parry with this shield. Maybe if you changed your grip?")
+			to_chat(user, "So heavy... You doubt your ability to parry with this shield until you grow stronger.")
 		if(31 to 45)
-			to_chat(user, "Holding this feels a little clumsy.")
+			to_chat(user, "Holding this feels a little clumsy. Perhaps you should hit the gym.")
 		if(46 to 55)
-			to_chat(user, "With a comfortable grip, you feel condifent in your ability to parry with this shield.")
+			to_chat(user, "A bit hefty, but you feel confident in your ability to parry with this shield.")
```

Comment:
At this point most shields are fairly powerful and the player likely has 30 to 60 rob

Not sure hefty is a fitting adjective

Reply:
To get to this level with a buckler (which seemed to be easier to proc higher examines) I needed 75 ROB.

To get the third tier I needed 130 rob.

Reply:
> 75 rob

This means you were already godlike. And you describe someone godlike to feel a buckler as "hefty". Weight is misleading, large shields are "heavier" yet get higher block chance with less rob